src/test/java/org/folio/bulkops/service/OpenCSVConverterTest.java

```diff
@@ -255,8 +267,10 @@ private static BulkOperationsEntity getEmptyBean(Class<BulkOperationsEntity> cla
       bean = new User();
     } else if (clazz.equals(Item.class)) {
       bean = new Item().withVersion(1);
-    } else {
+    } else if (clazz.equals(HoldingsRecord.class)) {
       bean = new HoldingsRecord().withVersion(2);
+    } else {
+      bean = new Instance();
```

Comment:
It makes sense to add additional verification and  throw UnsupportedOperationException if bean class is not appropriate.